packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/RestaurantRating.vue

```diff
@@ -54,18 +54,26 @@
                 {{ ratingsMax }}
             </data>
 
-            <!-- Number of ratings/Own rating message -->
-            <span aria-hidden="true">
+            <!-- Number of ratings -->
+            <span
+                v-if="count && !isOwnRating"
+                aria-hidden="true">
                 &#40;
                 <data
-                    v-if="!isOwnRating"
                     data-test-id="rating"
                     :class="[$style['c-restaurantCard-rating-count']]"
                     :value="count">
                     {{ count }}
                 </data>
+                &#41;
+            </span>
+
+            <!-- Own rating message -->
+            <span
```

Comment:
I wonder we could simplify the logic a bit if we swop this 2 blocks like this:
```
<span v-if="isOwnRating"> ... </span>

<span v-else-if="count"> ... </span>
```

Reply:
that's much cleaner - will update now! thanks :)